.gitignore

```diff
@@ -24,9 +24,14 @@ examples/*/lightning_logs/
 examples/*/demo*/
 examples/action_dann_lightn/configs_xianyuan/
 
+# Logs
+log-*.txt
```

Comment:
Another option is to do a clean up: https://fastai1.fast.ai/dev/test.html#after-test-cleanup
Up to you which is better.

Reply:
Test outputs are currently torn down after this line:

https://github.com/pykale/pykale/blob/e3d0329f34a952ac5b81514fe2829e075e244d31/tests/utils/test_logger.py#L13

I added logs to `.gitignore` for convenience when running the code normally. The tests should not leave files lying around.

Reply:
I don't know that before but now I learned. Thanks. Is this the official doc? : https://docs.pytest.org/en/stable/fixture.html#yield-fixtures-recommended